ci-scripts/JenkinsFile-OAI-Container-GitHub

```diff
@@ -159,8 +159,8 @@ pipeline {
                   // On the daily regression build, we will regenerate the base image and the master image
                   // Making sure that if the build fails for any reason we can go back to the previous versions
                   if (params.REGRESSION_TEST) {
-                    sh('docker image tag magma-dev-mme:ci-base-image magma-dev-mme:ci-base-image-prev')
-                    sh('docker image tag magma-dev-mme:master magma-dev-mme:master-prev')
+                    sh('docker image tag magma-dev-mme:ci-base-image magma-dev-mme:ci-base-image-prev || true')
```

Comment:
Why `|| true`? Should this not fail if the tagging fails?

Reply:
Honestly, all this `re-tagging` is in case the build fails. 
So I really want to keep running, even if we switch CI servers (like we did last week),